Formula/operator-sdk.rb

```diff
@@ -48,7 +52,14 @@ def install
       assert_match commit_regex, version_output
     end
 
-    output = shell_output("#{bin}/operator-sdk init --domain=example.com --license apache2 --owner BrewTest 2>&1", 1)
-    assert_match "failed to initialize project", output
+    mkdir "test"
+
+    chdir "test" do
```

Comment:
```suggestion
    mkdir "test" do
```

Reply:
updated in commit https://github.com/Homebrew/homebrew-core/pull/100353/commits/5a4283ab2399d369e2d3b942c946d4aa348e099a